client/http-client/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "sourceRoot": "src",
+    "rootDir": ".",
+    "outDir": "./out",
+    "baseUrl": "./src",
+  },
+  "include": ["./src/**/*", "./*.ts"],
```

Since we use `createAggregateError` inside of the package:

```suggestion
  "include": ["./src/**/*", "./*.ts"],
  "references": [{ "path": "../common" }],
```